src/aggregator/aggregator/leader_flush_mgr_test.go

```diff
@@ -296,137 +296,21 @@ func TestLeaderFlushManagerPrepareNoFlushNoPersist(t *testing.T) {
 		doneCh          = make(chan struct{})
 	)
 
-	placementManager := NewMockPlacementManager(ctrl)
-	placementManager.EXPECT().Shards().Return(shard.NewShards(nil), nil)
-
 	opts := NewFlushManagerOptions().SetJitterEnabled(false)
 	mgr := newLeaderFlushManager(doneCh, opts).(*leaderFlushManager)
 	mgr.nowFn = nowFn
-	mgr.placementManager = placementManager
-
 	buckets := testFlushBuckets(ctrl)
 	mgr.Init(buckets)
 
 	now = time.Unix(59, 100000000)
 	mgr.lastPersistAtNanos = now.UnixNano()
 	flushTask, dur := mgr.Prepare(buckets)
+	mgr.UpdateFlushTimes(buckets)
 	require.Nil(t, flushTask)
 	require.Equal(t, 900*time.Millisecond, dur)
 	require.Equal(t, 0, storeAsyncCount)
 }
 
-func TestLeaderFlushManagerPrepareNoFlushWithPersistOnce(t *testing.T) {
```

Comment:
All of these tests were batching up persists until a particular timing was hit; now that persists are refactored to run independently of a Run, they are not necessary

Reply:
Re-added after bringing back the flushTimesPersistEvery option